cif_pow.dic

```diff
@@ -7643,73 +8033,178 @@ save_pd_qpa_ext_std.phase_id
-    be recorded using other PD_QPA_EXT_STD data items
+    This category gives the value of the intensity or scale factor which is
+    divided by the corresponding calibration factor in order to allow
+    quantitative phase analysis to be undertaken. Further normalisation may be
+    necessary, and can be indicated.
+
+    The supported methodologies are enumerated in _pd_qpa_overall.method.
+
+    For a review on quantitative phase analysis, see Chapter 3.9 of
+    International Tables, Vol. H, and references therein.
 ;
-    _name.category_id             pd_qpa_ext_std
-    _name.object_id               special_details
-    _type.purpose                 Describe
-    _type.source                  Recorded
+    _name.category_id             PD_GROUP
+    _name.object_id               PD_QPA_INTENSITY_FACTOR
+
+    loop_
+      _category_key.name
+         '_pd_qpa_intensity_factor.diffractogram_id'
+         '_pd_qpa_intensity_factor.phase_id'
+
+save_
+
+save_pd_qpa_intensity_factor.diffractogram_id
+
+    _definition.id                '_pd_qpa_intensity_factor.diffractogram_id'
+    _definition.update            2023-01-15
+    _description.text
+;
+    The diffractogram (see _pd_diffractogram.id) to which the intensity factor
+    relates.
+;
+    _name.category_id             pd_qpa_intensity_factor
+    _name.object_id               diffractogram_id
+    _name.linked_item_id          '_pd_diffractogram.id'
+    _type.purpose                 Link
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_qpa_intensity_factor.phase_id
+
+    _definition.id                '_pd_qpa_intensity_factor.phase_id'
+    _definition.update            2023-01-22
+    _description.text
+;
+    The phase (see _pd_phase.id) to which the intensity factor applies.
+;
+    _name.category_id             pd_qpa_intensity_factor
+    _name.object_id               phase_id
+    _name.linked_item_id          '_pd_phase.id'
+    _type.purpose                 Link
+    _type.source                  Related
     _type.container               Single
     _type.contents                Text
 
 save_
 
-save_PD_QPA_INT_STD
+save_pd_qpa_intensity_factor.value
+
+    _definition.id                '_pd_qpa_intensity_factor.value'
+    _definition.update            2023-01-22
+    _description.text
+;
+    An intensity or scale factor value associated with the given phase and
+    diffractogram, which, when divided by the appropriate calibration value
+    defined in _pd_qpa_calib_factor.*, allows quantitative phase analysis to be
+    undertaken.
+
+    This value is not, in general, transferable between different program types
+    and versions, as each software package may incorporate different constants
+    or normalisations into their calculations. However, if all values for a
+    given diffractogram, or CIF container, are self-consistent, then
```

Comment:
Not sure what is meant by "CIF container" here. Is the intended meaning that there is a collection of diffractograms? What would be lost by dropping this?

Reply:
I think I was trying to say that if I had a bunch of diffractograms (eg in situ), and they all were analysed in a self-consistent manner, then I can compare the `_pd_qpa_intensity_factor.value` between all of the phases. As the sentence is written, "or CIF container," can be removed without any loss of meaning.